e2e/parallel/nftsFlow.test.ts

```diff
@@ -131,27 +131,6 @@ describe('Visit NFTs Gallery and Details Pages', () => {
     expect(expirationValueText !== 'Invalid Date');
   });
 
-  it('should be able to sort nfts alphabetically', async () => {
```

Comment:
sort has been moved to backend, so if anything we'd include this in an integration test. We don't want CI to fail because of an issue on simplehash's side.